### ISSUE TITLE
Add PowerShell jobs support

### ISSUE THREAD

**Issue by daviwil** [PowerShell/PowerShellEditorServices#193]
The PowerShellContext should be able to run PowerShell jobs without issue.  A user of the VS Code extension has reported that they're not able to run a script that starts a job.  More info here: https://github.com/PowerShell/vscode-powershell/issues/120

**Comment by daviwil** [comment]
Reopening, I think the vscode-powershell bug number slipped into the commit :)

**Comment by rkeithhill** [comment]
Yeah, sorry about.  I originally put the VSCode issue number in the commit message for 240 and started the PR but I changed the comment to the correct issue number before submitting the PR.  I was hoping that would correct things but apparently not.  :-(

**Comment by daviwil** [comment]
No problem at all, man.  This is the problem with having two repos for one effective product ;)

**Comment by tylerl0706** [comment]
I believe this is fixed now. However, there's one open comment in the other issue we need to hear about.
